tests/test_lint.py

```diff
@@ -211,35 +210,32 @@ def test_assert_no_leading_trailing_whitespace():
 
 
 def test_assert_model_parameters_in_condition_or_parameter_table():
-    document = libsbml.SBMLDocument(3, 1)
-    model = document.createModel()
-    model.setTimeUnits("second")
-    model.setExtentUnits("mole")
-    model.setSubstanceUnits('mole')
-    sbml.add_global_parameter(model, 'parameter1')
-    sbml.add_global_parameter(model, 'noiseParameter1_')
-    sbml.add_global_parameter(model, 'observableParameter1_')
+    import simplesbml
```

Comment:
Move to header? Currently imported multiple times.

Also imported within test cases in other files.

Reply:
The idea was not importing bulky libsbml everywhere, that when running non-sbml only, they will complete slightly faster (e.g., convenient when using auto-rerun).